thorin/dump.cpp

```diff
@@ -382,7 +382,7 @@ void World::dump(std::ostream& os) {
         auto dep    = DepTree(*this);
         auto dumper = Dumper(os, &dep);
 
-        for (const auto& import : imported()) print(os, ".import {};\n", import);
+        for (auto [_, name] : driver().imports()) print(os, ".import {};\n", name);
```

Comment:
If possible, it would probably be nice to also dump loaded plugins as `.plugin`.

Reply:
good point. I will do that.